arch/arm/src/armv7-a/pgalloc.h

```diff
@@ -228,9 +197,7 @@ uintptr_t arm_physpgaddr(uintptr_t vaddr);
  *
  ****************************************************************************/
 
-#ifdef CONFIG_ARCH_PGPOOL_MAPPING
```

Comment:
should we select ARCH_PGPOOL_MAPPING for cortex-a in Kconfig automatically?

Reply:
>should we select ARCH_PGPOOL_MAPPING for cortex-a in Kconfig automatically?

@xiaoxiang781216 
I think it's a good idea but should be done in a separate PR, because RISC-V has the same implementation.

Reply:
Ok, let's wait your new PR.